worker.js

```diff
@@ -0,0 +1,56 @@
+var Sway = require('sway')
```

Comment:
nice! If it's bringing in entire dependency tree into this file then it should work. This was something I wanted to do for while but never had time. Thanks!